Vagrantfile

```diff
@@ -8,4 +8,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "redash/dev"
   config.vm.synced_folder "./", "/opt/redash/current"
   config.vm.network "forwarded_port", guest: 5000, host: 9001
+  config.vm.provision "shell" do |s|
+    s.inline = "/opt/redash/current/install_requirements.sh"
```

Comment:
We wanted the install_requirements.sh to be the same as what is used in circle.yml, so that there isn't a configuration drift (where circle.yml is updated but the vagrant provisioning file isnt). I guess we could still keep this file separate and call it from both circle.yml and vagrant_provision.sh. What do you think?